### DISCUSSION TITLE
SelectList disable  specific option

### DISCUSSION THREAD

**Discussion by PavloParafiloCricut** [Megabit/Blazorise#4043] [Q&A]
How to disable 1 option in SelectList? 
![image](https://user-images.githubusercontent.com/96420517/184488326-b3385f3d-f2b0-459f-914f-bdba60d12800.png)

```
<SelectList TItem="MySelectModel"
            TValue="int"
            Data="@myDdlData"
            TextField="@((item)=>item.MyTextField)"
            ValueField="@((item)=>item.MyValueField)"
            SelectedValue="@selectedListValue"
            SelectedValueChanged="@MyListValueChangedHandler"
Disable/Readonly = "@((item)=>item.IsDisabled)"
            DefaultItemText="Choose your country" />
```

Ex: is it any param that accepts functions to determine if item should be disabled, like Disable/Readonly = "@((item)=>item.IsDisabled)" ?

**Comment by stsrki** [comment]
At the moment, we don't have such an option to disable an item based on a condition. The only other way would be to construct it manually by using the Select and SelectItem components.

**Reply by PavloParafiloCricut** [reply]
Thanks for answer. Any plans to implement it? It's pretty common functionality

**Reply by stsrki** [reply]
We can implement it but considering that we have little time for v1.1 we can only do it in v1.2. But, we're always open to contribution PRs.
